public/src/js/models/collections/collection.js

```diff
@@ -509,6 +509,24 @@ define([
         }
     };
 
+    Collection.prototype.lastAlertSent = function () {
```

Comment:
I think `lastAlertSent` should be a `ko.pureComputed` otherwise it won't update when you send an alert